azuredeploy.json

```
@@ -330,14 +348,18 @@
     "deliveryPrincipalResourceId": {
       "value": "[concat(subscription().id, '/resourceGroups/',resourceGroup().name,'/providers/Microsoft.ManagedIdentity/userAssignedIdentities/',parameters('deliveryIdName'))]",
       "type": "string"
-  },
-  "workflowPrincipalResourceId": {
+    },
+    "workflowPrincipalResourceId": {
       "value": "[concat(subscription().id, '/resourceGroups/',resourceGroup().name,'/providers/Microsoft.ManagedIdentity/userAssignedIdentities/',parameters('workflowIdName'))]",
       "type": "string"
-  },
-  "droneSchedulerPrincipalResourceId": {
+    },
+    "ingestionPrincipalResourceId": {
+      "value": "[concat(subscription().id, '/resourceGroups/',resourceGroup().name,'/providers/Microsoft.ManagedIdentity/userAssignedIdentities/',parameters('ingestionIdName'))]",
```

```suggestion
      "value": "[resourceId('Microsoft.ManagedIdentity/userAssignedIdentities', parameters('ingestionIdName'))]",
```

I know the rest of them were the way you had it as well, but we should rarely build resource IDs via the `concat()` function, that's a discouraged practice.